source/designers-guide/modify-jquery-plugins/index.md

```diff
@@ -0,0 +1,99 @@
+---
+layout: default
+title: Override jQuery plugins 
+github_link: designers-guide/modify-jquery-plugins/index.md
+shopware_version: 5.0.2
+indexed: true
+---
+
+## Introducing
+The storefront of the Shopware 5 with it's new Responsive Theme relies massively on jQuery plugins to provide a rich interface with a great usabilty. The plugins are optimized for the Responsive Theme but what's with custom themes, where we have to alter the functionality of a certain functionality. All of our jQuery Plugins are built upon a ```publish / subscribe``` pattern which makes it easy to add new functionality to existing plugins, but sometimes you need more. You want to override certain methods in a plugin to match your interface behaviors which can't be configure or accomplish using events.
```

Comment:
For inline code just use a single backtick instead of the tripple one.
The tripple is for codeblocks.